test/e2e/presto_prometheus_connector_up_test.go

```diff
@@ -0,0 +1,50 @@
+package e2e
+
+import (
+	"bytes"
+	"github.com/operator-framework/operator-metering/test/reportingframework"
+	"github.com/stretchr/testify/require"
+	"os"
+	"os/exec"
+	"strings"
+
+	"testing"
+)
+
+func testPrometheusConnectorWorks(t *testing.T, rf *reportingframework.ReportingFramework) {
+	t.Helper()
+	out, err := exec.Command(
+		"/usr/local/bin/kubectl",
+		"-n", rf.Namespace,
+		"get", "pods",
+		"-l", "app=presto",
+		"-o", "name",
+	).Output()
+	require.Nil(t, err, "expected querying for the presto pod name would produce no error")
+	require.NotEmpty(t, string(out), "unable to parse presto pod name")
+	// we know the output is going to be of the form `pod/<presto coordinator pod name>` so split by '/'
+	tmp := strings.Split(string(out), "/")[1]
+	host := strings.TrimSuffix(tmp, "\n")
+	cmd := exec.Command(
+		"/usr/local/bin/kubectl",
+		"-n", rf.Namespace,
+		"exec",
+		host,
+		"-c", "presto",
+		"--",
+		"presto-cli",
+		"--server", "https://presto:8080",
+		"--user", "root",
+		"--catalog", "prometheus",
+		"--schema", "default",
+		"--keystore-path", "/opt/presto/tls/keystore.pem",
+		"--execute",
+		"SELECT * FROM prometheus.default.up where timestamp > (NOW() - INTERVAL '10' second)",
+	)
+	var queryResults bytes.Buffer
+	cmd.Stderr = os.Stderr
+	cmd.Stdout = &queryResults
+	err = cmd.Run()
+	queryResultsStr := string(queryResults.Bytes())
+	require.Containsf(t, queryResultsStr, "endpoint=metrics", "query output didn't contain `endpoint=metrics`")
```

Comment:
I think we should just pass the `string(queryResultsStr.Bytes())` directly to `require` (like we do on line 26) w/o declaring a variable.